tests/dummy/app/routes/application.js

```diff
@@ -1,17 +1,16 @@
 import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
 
-export default Route.extend({
-  esriLoader: service('esri-loader'),
+export default class ApplicationRoute extends Route {
+  @service esriLoader
 
-  renderTemplate: function () {
```

Comment:
`renderTemplate` is now deprecated: https://deprecations.emberjs.com/v3.x/#toc_route-render-template

This wasn't causing problems for consuming apps, but it seemed like an easy fix.